src/packages/cli/src/Studio.ts

```
@@ -153,11 +153,14 @@ export class Studio implements Command {
 
     const serverUrl = `http://localhost:${port}`
     if (!browser || browser.toLowerCase() !== 'none') {
-      await open(serverUrl, {
-        app: browser,
-        url: true,
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
-      }).catch(() => { }) // Ignore any errors
```

This is not going to ignore errors like I expect